src/config/bootloader.rnc

```diff
@@ -104,7 +104,8 @@ section =
     element remap { "true" | "false" }? &
     element configfile {text}? &
     element image   {text}? &
-    element description {text}?
+    element description {text}? &
+    element nounzip {text}?
```

Comment:
Was this actually already dropped from master? I could not find it: https://github.com/search?q=nounzip+user%3Ayast&type=Code